pkg/view/component/timestamp.go

```diff
@@ -27,12 +27,21 @@ type TimestampConfig struct {
 
 // NewTimestamp creates a timestamp component
 func NewTimestamp(t time.Time) *Timestamp {
-	return &Timestamp{
-		Base: newBase(TypeTimestamp, nil),
-		Config: TimestampConfig{
-			Timestamp: t.Unix(),
-		},
+
+	actualTime, err := time.Parse(time.RFC1123 ,t.String())
+    
+	if err != nil {
+		return &Timestamp{
+			Base: newBase(TypeTimestamp, nil),
+			Config: TimestampConfig{
+				Timestamp: t.Unix(),
+			},
+		}
+	} else {
+		
+		// Need to return something if it is not of proper format
```

Comment:
Should we return a zero time here or a null value

Reply:
I wouldn't worry about this part as the behavior of timestamp component is as expected, and the issue is about not passing bad inputs.

Reply:
Understood :+1: